boards/adi/max32690evkit/doc/index.rst

```diff
@@ -303,7 +303,7 @@ instead of ``west flash``.
 References
 **********
 
-- `MAX32690EVKIT web page`_
+- `MAX32690EVKIT solution center`_
 
-.. _MAX32690EVKIT web page:
-   https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/MAX32690EVKIT.html
+.. _MAX32690EVKIT solution center:
+   https://developer.analog.com/solutions/max32690
```

Comment:
I think it make sense to address developer portal.
It requires one more click to reach the board page, but it is fine for me.

This update should be propagated to MAX32690FTHR and apard32690 board too.

Reply:
Thank you for reviewing and evaluating the documentation link request. I'll also update the other pages as well. Disclaimer: Still learning here - tips and advice appreciated!